cvmfs/download.cc

```diff
@@ -177,25 +177,27 @@ static size_t CallbackCurlHeader(void *ptr, size_t size, size_t nmemb,
   //          header_line.c_str());
 
   // Check http status codes
-  if (HasPrefix(header_line, "HTTP/1.", false)) {
-    if (header_line.length() < 10)
+  if (HasPrefix(header_line, "HTTP/", false)) {
+    if (header_line.length() < 12) {
```

Comment:
Huh? No... it is okay how it is now all for `HTTP/1.0`, `HTTP/1.1` and `HTTP/2.0`! Note that H2 also sends the minor version number in the header and there must be a minimum of **12** characters for a correct first header line including an HTTP status code:

```
example:
HTTP/2.0 200

 1: H
 2: T
 3: T
 4: P
 5: /
 6: 2
 7: .
 8: 0
 9:  
10: 2
11: 0
12: 0
```